front/src/applications/operationalStudies/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -258,155 +262,269 @@ const createTrainSchedulePayload = async ({
     };
   });
 
+/**
+ * Generate properties for trainrun related caracteristics.
+ */
+const generateTrainrunProperties = async (
+  netzgrafikDto: NetzgrafikDto,
+  trainrun: TrainrunDto,
+  infraId: number,
+  dispatch: AppDispatch,
+  startTime?: string
+) => {
+  const trainrunSections = getTrainrunSectionsByTrainrunId(netzgrafikDto, trainrun.id);
+  const path = await generatePath(trainrunSections, netzgrafikDto.nodes, infraId, dispatch);
+  const trainrunLabels = compact(
```

Comment:
Nit: maybe this could be renamed to `labels` so that callers don't need to specify the field name when populating the timetable item payload for editoast?